src/main/java/com/example/petbuddybackend/entity/photo/CloudPhoto.java

```diff
@@ -0,0 +1,29 @@
+package com.example.petbuddybackend.entity.photo;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.Id;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(of = "blob")
+@EntityListeners(CloudPhotoDeleteListener.class)
+public class CloudPhoto {
```

Comment:
w sumie to moze nazwac to jakos PhotoLink, albo PhotoData. Bo to nie jest zdjecie, tylko informacje jego lokalizacji w firebase z tego co rozumiem

Reply:
Tak, klasa ma: 
- blob - nazwa pliku w firebase
- url
- urlExpiresAt 

urlExpiresAt jest po to bo te linki nie są wieczne i trzeba je odnawiać więc "cachuje" url w bazie danych i w razie potrzeby odnawiam

Reply:
A nazwę można zmienić